### ISSUE TITLE
transition bug

### ISSUE THREAD

**Issue by chrisdrackett** [Compass/compass#805]
I just updated compass and I've run into the following:

I have my transition properties defined in a variables file:

```
$default-transition-property : all          !default
$default-transition-duration : .15s         !default
$default-transition-function : ease-in-out  !default
$default-transition-delay    : false        !default
```

then I generally just add `+transition()` to a class that I want to animate. However, I see that this now outputs:

```
-webkit-transition: all, 0.3s, ease-in-out;
-moz-transition: all 0.3s ease-in-out;
-ms-transition: all 0.3s ease-in-out;
-o-transition: all 0.3s ease-in-out;
transition: all, 0.3s, ease-in-out;
```

I believe that the commas are causing issues. Maybe I'm missing something?

**Comment by chriseppstein** [comment]
@chrisdrackett what version of compass?

**Comment by chrisdrackett** [comment]
0.12.1

**Comment by scottdavis** [comment]
werid

**Comment by scottdavis** [comment]
@doctyper was the last one to work in this file maybe he can comment?

**Comment by chriseppstein** [comment]
I want to do a 0.12.2 release this weekend. This needs to be addressed for that release.

**Comment by doctyper** [comment]
Bump @chriseppstein. Don't want to miss your .2 release.

**Comment by mirisuzanne** [comment]
closed by 0c13d98cb40d13edafa2f646e9656ac5546b3f80
